configure.ac

```diff
@@ -1054,8 +1054,8 @@
             [enable_af_packet="no"],
             [[#include <sys/socket.h>
               #include <linux/if_packet.h>]])
-        AC_CHECK_DECL([PACKET_FANOUT],
```

Comment:
No, see https://github.com/regit/suricata/blob/af-packet-new-modes-v1.2/src/source-af-packet.h#L27, we are forcing the declaration if the headers do not have the define.